# HG changeset patch
# User Rik <rik@octave.org>
# Date 1363974180 25200
#      Fri Mar 22 10:43:00 2013 -0700
# Node ID 0db0926c2d0f4342b2a6545ca2cf66704afd9fa7
# Parent  0cbe330f39a2033783d4265a5e508d90bfcf1ab8
maint: update gnulib for upstream fix for bug #38424

* .hgsubstate: update gnulib for upstream fix for bug #38424

diff --git a/.hgsubstate b/.hgsubstate
--- a/.hgsubstate
+++ b/.hgsubstate
@@ -1,1 +1,1 @@
-cec099cbf54f2d3a49367c03df101f3cf35c20c7 gnulib-hg
+5a51fb7777a9950502965a043a70bd6ca5e0498b gnulib-hg
